src/components/image/index.jsx

```diff
@@ -76,6 +76,7 @@ const Image = ({
 	return (
 		<img
 			{...rest}
+			data-chromatic="ignore"
```

Comment:
I only worry that this will completely ignore changes, but since we go through design review maybe this is ok?

Reply:
I have the same worry - but we are seeing a huge amount of false positives with chromatic runs and it's due to image changes within the image, sub pixel rendering/quality. We can also undo this change if it starts to miss too much